cloudsync/api.py

```diff
@@ -385,7 +391,9 @@ def upload_subtitle_to_s3(caption_data, file_data):
         try:
             vt.delete_from_s3()
         except ClientError:
-            log.exception('Could not delete old subtitle from S3: %s', vt.s3_object_key)
+            log.exception('Could not delete old subtitle from S3: %s',
+                          positional_args=(vt.s3_object_key,),
+                          s3_object_key=vt.s3_object_key)
```

Comment:
`s3_object_key` is a tag so no need for positional_args here:

```python
log.exception('Could not delete old subtitle from S3',
                          s3_object_key=vt.s3_object_key)
```

Reply:
Changed in 17e0a9e